test/acceptance/controller.js

```diff
@@ -989,6 +989,29 @@ describe('Controller', function () {
         })
     })
 
+    it('should return pre and post image details', function (done) {
+      var newTestConfig = JSON.parse(testConfigString)
+      newTestConfig.images.directory.enabled = true
+      newTestConfig.images.directory.path = './test/images'
+      fs.writeFileSync(config.configPath(), JSON.stringify(newTestConfig, null, 2))
+
+      config.loadFile(config.configPath())
+
+      var client = request(cdnUrl)
+      client
+        .get('/test.jpg?quality=100&width=180&height=180&resizeStyle=entropy&format=json')
+        .end(function (err, res) {
+          res.statusCode.should.eql(200)
+
+          res.body.fileSizePre.should.eql(173685)
+          res.body.fileSizePost.should.eql(54089)
```

Comment:
@philip-hunt the test is failing on Travis CI, I guess because it's processing it slightly differently:

`expected 53851 to equal 54089`

Maybe modify the test so it's not so specific:

```
let originalSize = res.body.fileSizePre
res.body.fileSizePost.should.be.below(originalSize)
```

Reply:
(likely the same for the colour test too!)